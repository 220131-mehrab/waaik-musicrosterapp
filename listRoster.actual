./musicrosterapp <command> [<args>]
Commands
  listRoster <role>         list of desired role and names
  numRoles   <role>           num count total number of roles
 <role> = singer, musician, producer    roles available
