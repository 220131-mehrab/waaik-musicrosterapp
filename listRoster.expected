./musicrosterapp <command> [<args>]
Commands
  listRoster <role>         list of desired role and names
  numRoles   <role>           num count total number of roles
 <role> = singer, musician, producer    roles available
Anthia,Goddard,musician
Antonietta,Kirbee,musician
Binny,Kamaria,musician
Brietta,Amand,musician
Charissa,Orpah,musician
Cyndie,Thomasina,musician
Daryl,Euridice,musician
Doro,Faria,musician
Elie,Bates,musician
Ellette,Girardo,musician
Eve,Janene,musician
Gretal,Mendez,musician
Hannis,Forrer,musician
Janenna,Hartnett,musician
Jasmina,Tengdin,musician
Judy,Douglass,musician
Kerrin,Cosenza,musician
Kimberley,Love,musician
Layla,Hoenack,musician
Lyssa,Ledah,musician
Marcelline,Tryck,musician
Marnia,Shuler,musician
Mary,Khorma,musician
Olwen,Couture,musician
Reeba,Ciro,musician
Roxane,Stevy,musician
Susan,Gwenore,musician
Tani,Swigart,musician
Theodora,Yam,musician
Tybie,Hourigan,musician
Ulrike,Chandler,musician
